reana/reana_dev/git.py

```diff
@@ -482,6 +482,52 @@ def git_checkout(branch, fetch):  # noqa: D301
             display_message(msg, component)
 
 
+@click.option(
+    "--branch", "-b", nargs=2, multiple=True, help="Which PR? [component PR#]"
+)
+@click.option("--push", is_flag=True, default=False)
+@git_commands.command(name="git-merge")
+def git_merge(branch, push):  # noqa: D301
+    """Merge to local master a component pull request.
+
+    The ``-b`` option can be repetitive to merge several pull requests in
+    several repositories at the same time.
+
+    \b
+    :param branch: The option ``branch`` can be repeated. The value consist of
+                   two strings specifying the component name and the pull
+                   request number. For example, ``-b reana-workflow-controler
+                   72`` will merge a local branch called ``pr-72`` from the
+                   reana-workflow-controller to master branch.
+    :param push: Should we push to origin and upstream? [default=False]
+    :type branch: list
+    :type push: bool
+    """
+    for cpr in branch:
+        component, pull_request = cpr
+        component = select_components([component,])[0]
+        if component in REPO_LIST_ALL:
+            for cmd in [
+                "git fetch upstream",
+                "git diff pr-{0}..upstream/pr/{0}".format(pull_request),
+                "git checkout master",
+                "git merge --ff-only upstream/master",
+                "git merge --ff-only upstream/pr/{0}".format(pull_request),
+                "git branch -D pr-{0}".format(pull_request),
```

Comment:
Is `-d` working here? Would be better.

(If `pr-n` is equal to `upstream/pr/n`, which is ensured above, then `git branch -d` should be working... Using `-D` is dangerous if some remote updates or local updates get in.)